src/Resources/views/Search/_paging.html.twig

```diff
@@ -0,0 +1,12 @@
+<span class="paging-help">{{ pagingHelp.start }}-{{ pagingHelp.end }} sur {{ pagingHelp.total }}</span>
```

Comment:
Could you add a translation somewhere?

You can use placeholders in your translation, like `%start%` and `%end%`…